### PR TITLE
(Feature, DEMO) Add ValidatorMetadata to drop-down list in "Modify Proxy Contract Ballot"

### PR DESCRIPTION
**Pull request by varasev**
**(Mandatory) Description**

These changes correspond to the new version of ProxyStorage contract from https://github.com/poanetwork/poa-network-consensus-contracts/pull/66 PR.

Also it includes small fixes described in https://github.com/poanetwork/poa-network-consensus-contracts/issues/45 and https://github.com/poanetwork/poa-dapps-voting/issues/117.

**(Mandatory) What is it: (Fix), (Feature), or (Refactor)**

(Feature)

### CODE REVIEW

File: src/stores/BallotStore.js [hunk 1]
```diff
@@ -22,7 +22,9 @@ class BallotStore {
 		2: 'VotingToChangeKeys',
 		3: 'VotingToChangeMinThreshold',
 		4: 'VotingToChangeProxy',
-		5: 'BallotsStorage'
+		5: 'BallotsStorage',
+		7: 'ValidatorMetadata',
+		8: 'ValidatorMetadataEternalStorage'
```
Comment (rstormsf) [inline src/stores/BallotStore.js:27]: I don't get it, can we change the storage? How?

**Comment by rstormsf** [comment]
What is the reason behind having 2 PRs for Demo and Core branches? Would not be easier to do 1 PR instead to Core branch?

File: src/stores/BallotStore.js (hunk 1 above)
Comment (varasev) [reply src/stores/BallotStore.js:27]: Yes, we can change the address of **ValidatorMetadata** storage. See **setContractAddress** function in **ProxyStorage.sol**. This function is called by **finalizeBallot** in **VotingToChangeProxyAddress.sol**.

**Comment by varasev** [comment]
Yes, but the demo branch should be updated too, isn't it? Or you will merge the branch?

**Review by vbaranov — APPROVED** [review]

**Pull request merged**